src/main/java/edu/kis/powp/jobs2d/drivers/adapter/Jobs2DToDrawAdapter.java

```diff
@@ -4,16 +4,18 @@
 import edu.kis.legacy.drawer.shape.ILine;
 import edu.kis.legacy.drawer.shape.LineFactory;
 import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
 
 /**
  * driver adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class Jobs2DToDrawAdapter extends DrawPanelController implements Job2dDriver {
 	private int startX = 0, startY = 0;
-
-	public MyAdapter() {
+  private final DrawPanelController drawer;
+	public Jobs2DToDrawAdapter(DrawPanelController drawPanelController) {
```

Comment:
Original code used tabs instead of spaces. It is important to keep the original convention. It makes tracking actual changes more difficult.